provisioning/provisioning-service-client/src/test/java/com/microsoft/azure/sdk/iot/provisioning/service/configs/IndividualEnrollmentTest.java

```diff
@@ -228,12 +228,12 @@ public void constructorWithJsonSetsOptionalParametersUsesSetters()
 
         // assert
         assertEquals(VALID_DEVICE_ID, Deencapsulation.getField(enrollment, "deviceId"));
-        assertNotNull(Deencapsulation.getField(enrollment, "registrationState"));
+        assertNotNull(enrollment.getDeviceRegistrationState());
```

Comment:
why was this change required?

Reply:
It wasn't required, but the method could be accessed directly rather than through reflection since the field is package private, not private. If we ever refactor the getDeviceRegistrationState() method, this unit test would get updated for free